From fc1a4d96f621d3f178ec4872ad2bc740413467ed Mon Sep 17 00:00:00 2001
From: Florian Uhlig <f.uhlig@gsi.de>
Date: Fri, 16 Jan 2015 17:53:53 +0100
Subject: [PATCH] Correct erroneous include statement which specified the
 complete path instead of the macro name. This results in an error when using
 CMake 3.1.0. Fix #53.

---
 cmake/modules/FindProtobuf.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindProtobuf.cmake b/cmake/modules/FindProtobuf.cmake
index 225de1e..a97a872 100644
--- a/cmake/modules/FindProtobuf.cmake
+++ b/cmake/modules/FindProtobuf.cmake
@@ -235,7 +235,7 @@ find_program(PROTOBUF_PROTOC_EXECUTABLE
 mark_as_advanced(PROTOBUF_PROTOC_EXECUTABLE)
 
 
-include(${CMAKE_MODULE_PATH}/FindPackageHandleStandardArgs.cmake)
+include(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(PROTOBUF DEFAULT_MSG
     PROTOBUF_LIBRARY PROTOBUF_INCLUDE_DIR)
 
